services/reflex/knowledge_base_construction/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: "2.3"
+services:
+    lm:
+        build: .
+        #image: reflex:latest
+        ipc: host
+        volumes:
+            - ./src/:/kbc/
+            - /ssd/hadar/data:/data/
+            - /ssd/hadar/fairseq/fairseq:/kbc/fairseq
+            - /ssd/hadar/LAMA/lama:/kbc/lama
```

Comment:
1. Remove the LAMA dependency
2. Remove roberta connecter subclass (base connecter)
3. Refactor used functions